hooks/useCurrentSegmentNode.ts

```diff
@@ -4,22 +4,22 @@ import { SegmentNode, admin, main } from '@/types/page';
 
 export default function useCurrentSegmentNode(): SegmentNode {
   const pathname = usePathname();
-  if (pathname === `/${admin.segment}`) return admin; // admin은 main과 별도
   const segments = pathname.split('/');
   segments.shift(); // 맨 앞의 공백 제거
-  let node = main;
 
-  for (const segment of segments) {
-    let child = findChildWithName(node, segment);
-    if (!child) break;
-    node = child;
-  }
+  // 영어 버전일 경우 /en 제거
+  if (segments.length && segments[0] === 'en') segments.shift();
 
-  return node;
-}
+  // admin 예외처리
+  if (pathname === `/${admin.segment}`) return admin;
```

Comment:
얘 그대로 냅두면 `en/admin` 됐을 때 사소한 문제가 생길 듯...?! 어드민 페이지야 행정실에서 쓸 테니 영어로 번역할 필요는 없겠지만 어쨌거나 영어 상태에서 admin 들어갔을 때 서브내비에 관련 페이지들은 정상적으로 들어가 있는 게 좋을 것 같아서...! 
```suggestion
  if (segments[0] === admin.segment) return admin;
```